composer.json

```diff
@@ -10,7 +10,8 @@
         }
     ],
     "require": {
-        "php": ">=7.1"
+        "php": ">=7.1",
+        "illuminate/support": "^5.7"
```

Comment:
This should be `^5.4`, the library is supposed to work with Lumen >= 5.4

Reply:
I changed it to ^5.4